control-plane/subcommand/inject-connect/command.go

```diff
@@ -146,10 +149,16 @@ func (c *Command) init() {
 		"Directory with PEM-encoded TLS certificate and key to serve.")
 	c.flagSet.StringVar(&c.flagConsulImage, "consul-image", "",
 		"Docker image for Consul.")
+	c.flagSet.StringVar(&c.flagConsulImageWindows, "consul-image-windows", "",
+		"Windows Docker image for Consul.")
```

Comment:
Do we need this flag? We don't use the consul image for injection I think.

Reply:
We don't need this flag. I added it just for consistency. If you agree, I can remove it.

Reply:
Yep, let's remove it!